doc/release-notes-1.2.0.rst

```diff
@@ -0,0 +1,586 @@
+* The SEGGER Control Block cannot be found by automatic search by the RTT Viewer/Logger.
+  As a workaround, set the RTT Control Block address to 0 and it will try to search from address 0 and upwards.
+  If this does not work, look in the ``builddir/zephyr/zephyr.map`` file to find the address of the ``_SEGGER_RTT`` symbol in the map file and use that as input to the viewer/logger.
+* nRF91 fails to receive large packets (over 4000 bytes).
+* nrf_connect fails if called immediately after initialization of the device.
+  A delay of 1000 ms is required for this to work as intended.
+
+
+Subsystems
+==========
+
+Bluetooth Low Energy
+--------------------
+
+* Bluetooth LE cannot be used in a non-secure application, for example, an application built for the nrf5340_dk_nrf5340_cpuappns board.
+  Use the nrf5340_dk_nrf5340_cpuapp board instead.
+* The :ref:`peripheral_hids_keyboard` sample cannot be used with the :ref:`nrfxlib:ble_controller` because the NFC subsystem does not work with the controller library.
+  The library uses the MPSL Clock driver, which does not provide an API for asynchronous clock operation.
+  NFC requires this API to work correctly.
+* When the :ref:`peripheral_hids_mouse` sample is used with the Zephyr Bluetooth LE Controller, directed advertising does not time out and the regular advertising cannot be started.
+* The :ref:`bluetooth_central_hids` sample cannot connect to a peripheral that uses directed advertising.
+* Bluetooth Low Energy peripheral samples are unstable in some conditions (when pairing and bonding are performed and then disconnections/re-connections happen).
+* When running the :ref:`bluetooth_central_dfu_smp` sample, the :option:`CONFIG_BT_SMP` configuration must be aligned between this sample and the Zephyr counterpart (:ref:`zephyr:smp_svr_sample`).
+  However, security is not enabled by default in the Zephyr sample.
+* The central samples (:ref:`central_uart`, :ref:`bluetooth_central_hids`) do not support any pairing methods with MITM protection.
+* On some operating systems, the nrf_desktop application is unable to reconnect to a host.
+* central_uart: A too long 212-byte string cannot be handled when entered to the console to send to peripheral_uart.
+* On nRF51 devices, Bluetooth LE samples that use GPIO might crash when buttons are pressed frequently.
+  In such case, the GPIO ISR introduces latency that violates real-time requirements of the Radio ISR.
+  nRF51 is more sensitive to this issue than nRF52 (faster core).
+
+
+Bootloader
+----------
+
+* Building and programming the immutable bootloader (see :ref:`ug_bootloader`) is not supported in SEGGER Embedded Studio.
+* The immutable bootloader can only be used with the following boards:
+
+  * nrf52840_pca10056
+  * nrf9160_pca10090
+
+  It does not work properly on nRF51 and nRF53.
+
+
+NFC
+---
+
+* The NFC poller samples cannot be run on the nRF5340 PDK.
```

Comment:
NFC poller and NFC reader samples (:ref:)

Reply:
:ref:\`nfc_tnep_poller\`
:ref:\`nfc_tag_reader\`